metricshub-hardware/src/main/java/org/sentrysoftware/metricshub/hardware/strategy/HardwarePostDiscoveryStrategy.java

```diff
@@ -95,6 +114,11 @@ public void run() {
 			.map(Map::values)
 			.flatMap(Collection::stream)
 			.filter(monitor -> monitorHasKnownType(monitor.getType()))
+			.filter(monitor ->
+				(!monitor.getType().equals(KnownMonitorType.HOST.getKey()) &&

```

Comment:
Replace `monitor.getType().equals(KnownMonitorType.HOST.getKey()` with `monitor.isEndpointHost()`. And create a variable for `monitor.isEndpointHost()` in the filter function.